src/HealthGPS/static_linear_model.cpp

```diff
@@ -58,6 +71,15 @@ void StaticLinearModel::generate_risk_factors(RuntimeContext &context) {
 
     // Adjust risk factors to match expected values.
     adjust_risk_factors(context, names_);
+
+    // Initialise everyone.
+    for (auto &person : context.population()) {
+        // HACK: start intervening after 2 years from sim start.
```

Comment:
Out of curiosity, why 2 and not 1 or 3 or something else?

Reply:
I think this was somewhat arbitrarily chosen by Ali, possibly to 'settle down' the simulation before intervening. This should eventually be customised. I'll open an issue.

Reply:
#287